lib/msal-browser/docs/accounts.md

```diff
@@ -35,25 +57,79 @@ function handleResponse(resp) {
 }
 ```
 
-Now the `homeAccountId`, `localAccountId`, or `username` can be used to look up the cached account before acquiring a token silently:
+Now, account properties such as: `homeAccountId`, `localAccountId`, and `username` can be used to look up the cached account before acquiring a token silently:
 
 ```javascript
 // This method attempts silent token acquisition and falls back on acquireTokenPopup
-async function getTokenPopup(request, account) {
-    request.account = myMSALObj.getAccountByHomeId(homeAccountId); // alternatively: myMSALObj.getAccountByLocalId(localAccountId) or myMSALObj.getAccountByUsername(username) 
+async function getTokenPopup(request, homeAccountId) {
+    // In this case, accounts are filtered by homeAccountId, but more attributes can be added to refine the search and increase the precision of the account filter
+    const accountFilter = {
+        homeAccountId: homeAccountId,
+    };
+    request.account = myMSALObj.getAccountByFilter(accountFilter);
     return await myMSALObj.acquireTokenSilent(request).catch(async (error) => {
-       // Handle error
+        // Handle error
         return await myMSALObj.acquireTokenPopup(request);
     });
 }
 ```
 
+### Filtering by login hint
+
+As of `@azure/msal-browser@3.11.0`, all login hint values can be used to search for and filter accounts. In order to filter by login hint, MSAL will compare the `loginHint` value in the `AccountFilter` object against the following account attributes (in order of precedence) to search for matches:
```

Comment:
3.11?

Reply:
The next release, this will be a minor change so the 3.11 is going to be the next version.

Reply:
We're on 3.1 right now, next release will be 3.2

Reply:
Sorry my bad, misread :S